tools/landmark_registration_ls/landmark_registration_ls.xml

```diff
@@ -29,5 +30,9 @@
     **What it does**
 
     This tool estimates the (affine) transformation matrix using least squares given two sets of 2d coordinates of landmarks.
+
+    Note: This tool is deprecated and may not be supported in future. Please use the tool "Landmark Registration" instead.
```

Comment:
I can deprecate tools on EU. Can you ping me with this once we update EU?

Reply:
Thanks. But I would keep the deprecated tools for some time on EU before they are uninstalled.

Do you mean uninstallation?

Reply:
No, we never uninstall tools, but we can hide them. So no new workflows are created with them.

Reply:
This is cool. I might merge a few more tools in next days. Afterwords I will let you know which tools I would deprecated so that you only need to do the operation once. Thanks.